js/classes/entities/midground/chad.js

```diff
@@ -105,14 +114,19 @@ class Chad {
     static get RUN_MULTIPLIER() {
         return 1.4;
     }
-    /** The mulitiplier that allows CHAD to run. */
+    /** How much faster Chad is while dashing than walking */
     static get DASH_MULTIPLIER() {
         return 3.5;
     }
 
     /** The barrier that limits the length of the longest dash. */
-    static get DASH_BARRIER() {
-        return 250;
+    static get DASH_LIMIT() {
```

Comment:
What is dash limit? Is this the x distance that limits his dash or is this time for a timer? Perhaps we could specify what unit or direction in the name?